projects/prometheus/prometheus/helm/patches/0002-Add-node-exporter-component.patch

```diff
@@ -527,33 +527,55 @@ index 27dc52e8..701996da 100644
 -    ##
 -    image:
 -      repository: quay.io/prometheus-operator/prometheus-config-reloader
--      tag: v0.73.2
+-      tag: v0.75.2
 -      # When digest is set to a non-empty value, images will be pulled by digest (regardless of tag value).
 -      digest: ""
 -      pullPolicy: IfNotPresent
+-
+-    ## config-reloader's container port and port name for probes and metrics
+-    containerPort: 8080
+-    containerPortName: metrics
+-
+-    ## Additional configmap-reload container arguments
+-    ## Set to null for argumentless flags
+-    ##
+-    extraArgs: {}
 +  ## If true, node-exporter pods mounts host / at /host/root
 +  ##
 +  hostRootfs: true
  
--    # containerPort: 9533
+-    ## Additional configmap-reload volume directories
+-    ##
+-    extraVolumeDirs: []
 +  ## node-exporter container name
 +  ##
 +  name: node-exporter
  
--    ## Additional configmap-reload container arguments
--    ## Set to null for argumentless flags
+-    ## Additional configmap-reload volume mounts
 -    ##
--    extraArgs: {}
+-    extraVolumeMounts: []
 +  ## node-exporter container image
 +  ##
 +  image:
 +    repository: quay.io/prometheus/node-exporter
```

Comment:
This needs to be replaced and templated out in values.yaml or no? Or at least the subsequent patches need to be fixed to make sure there are no conflicts right?

Reply:
Yes, this is templated out and its done in patch 0003.
https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/prometheus/prometheus/helm/patches/0003-Update-image-repo.patch#L78